feature/bgp/feature.textproto

```diff
@@ -430,6 +430,18 @@ config_path {
 telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/timers/state/keepalive-interval"
 }
+config_path {
```

Comment:
In the end of this file, you would need a feature profile dependency on NetworkInstance base feature profile as you want the BGP protocol under network instance.